sql/base/realmd.sql

```diff
@@ -58,7 +58,9 @@ CREATE TABLE `account` (
   `expansion` tinyint(3) unsigned NOT NULL DEFAULT '0',
   `mutetime` bigint(40) unsigned NOT NULL DEFAULT '0',
   `locale` tinyint(3) unsigned NOT NULL DEFAULT '0',
-  `token` text,
+  `security` int(11) DEFAULT '0',
+  `pin` int(11) DEFAULT '0',
+  `token` varchar(16) DEFAULT NULL,
```

Comment:
Why this change right here? it was "text" before.

Reply:
Because varchar takes up less space, it's indexable and has a variable max size (so we can set it to 16)
Text on the other hand...

https://stackoverflow.com/questions/25300821/difference-between-varchar-and-text-in-mysql